### PR TITLE
Fix compiler binary pushes

### PR DESCRIPTION
**Pull request by rsimha**
Follow up to #6.

Two changes:
- The `github_token` option is not necessary. Removing it.
- The different jobs need to merge their binaries into one commit, so we must use `--force` and skip the dirty branch check.

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -42,5 +42,6 @@ jobs:
       if: ${{ github.event_name == 'push' }}
       with:
         commit_message: Push new compiler binaries
+        push_options: '--force'
         file_pattern: packages/google-closure-compiler-*/compiler*
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+        skip_dirty_check: true
```
Comment (kristoferbaxter) [inline .github/workflows/build-test.yml:47]: Should we move these independent platforms to different commits based on success of the previous? It would be a bit slower to run in succession, but the Github UI would show the correct status for each of the actions.

Comment (rsimha) [reply .github/workflows/build-test.yml:47]: Gotcha. I'll have to figure out how to do the testing in parallel, and after that, the pushing in series. Will merge this and try that in a separate PR once the repo is actually deployed.

Comment (rsimha) [reply .github/workflows/build-test.yml:47]: Done in #8.

**Comment by rsimha** [comment]
That didn't work. The three commits did go in one after another with different commit messages, but they kept replacing each other, and only the last one is now in the repo. I'll reevaluate the tooling we're using to generate these commits and come back with a new PR.
